torch/csrc/jit/codegen/cuda/expr_simplifier.cpp

```diff
@@ -16,6 +16,104 @@ namespace jit {
 namespace fuser {
 namespace cuda {
 
+namespace debug_print {
```

Comment:
I moved this namespace from bottom to top. Most of them are just code movements, the only related thing is the comment below, and the `shouldPrint`.